README.md

```diff
@@ -330,13 +296,13 @@ The above figure shows the K64F with the v1.0.0 CI test shield fitted. Note:
 
 1. Make sure `mbed-cli` is working correctly and its version is newer than `1.0.0`.
 
- ```
+ ```sh
  mbed --version
  ```
 
  If not, update it:
 
- ```
+ ```sh
  pip install mbed-cli --upgrade
  ```
  
```

Comment:
This is still Mbed CLI 1, even though we added instructions for both 1 and 2. I'm not sure we need this troubleshooting, because the "Mbed OS build tools" section near the top seems enough.